stdlib/pervasives.ml

```diff
@@ -0,0 +1,244 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                   Jeremie Dimino, Jane Street Europe                   *)
+(*                                                                        *)
+(*   Copyright 2017 Jane Street Group LLC                                 *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+[@@@deprecated "Use Stdlib instead"]
```

Comment:
How do I trigger this warning?

I tried referencing values from pervasives (e.g. `let _ = Pervasives.exp 1.0`) with `-w A`, but did not get any warning.

Reply:
Indeed, it seems that this attribute is silently ignored by the compiler. Do you know where we are supposed to put it?

Reply:
The following works, but requires a nested module:
```
module M = struct let x = 12 end [@@deprecated "foo"]
```

Reply:
It has to do with module aliases; if one uses the long name (eg `Stdlib__pervasives.exp`), then the warning is correctly triggered.

Reply:
I see, I'll try to put the attribute on the alias in stdlib.mli then

Reply:
Adding it to the alias in stdlib.mli worked. In particular it caught all the cases in the compiler itself